pipelinewise/cli/pipelinewise.py

```diff
@@ -1613,12 +1614,24 @@ def import_project(self):
         total_targets = 0
         total_taps = 0
         discover_excs = []
+        found_selected_taps = set()
 
         # Import every tap from every target
         start_time = datetime.now()
         for target in config.targets.values():
             total_targets += 1
-            total_taps += len(target.get('taps'))
+            selected_taps = []
+
+            if selected_taps_id == ['*']:
+                total_taps += len(target.get('taps'))
+                selected_taps = target.get('taps')
+
+            else:
+                total_taps = len(selected_taps_id)
```

Comment:
This variable will be set in every iteration through the targets, I think it's better to move it outside the `for target in config.targets.values():` loop

Reply:
- [ ] Fix it!